src/main/flight/pid.h

```diff
@@ -246,3 +255,7 @@ void dynLpfDTermUpdate(float throttle);
 void pidSetItermReset(bool enabled);
 float pidGetPreviousSetpoint(int axis);
 
+
+extern float dT;
```

Comment:
Not too keen on having them exposed as externs - defining getters would make the purpose clearer, and will be inlined to direct access anyway.

Reply:
done